src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_eventhub_commands_eventhub_test.py

```diff
@@ -45,7 +45,7 @@ def test_eh_eventhub(self, resource_group):
 
         # updated teh Storageaccount ID
         subid = self.cmd('account show --query id -otsv').output.replace('\n', '')
-        storageaccountid = '/subscriptions/' + subid + '/resourcegroups/v-ajnavtestcli/providers/Microsoft.Storage/storageAccounts/testing1sdkeventhub'
+        storageaccountid = '/subscriptions/' + subid + '/resourcegroups/v-ajnavtest/providers/Microsoft.Storage/storageAccounts/testingsdkeventhubnew'
```

Comment:
I was wondering why you are using fixed 'resource-group' name and 'storage-account' name? Is it possible to create a storage account before using?